release-notes/5.16.1.md

```diff
@@ -0,0 +1,38 @@
+# CiviCRM 5.16.1
+
+Released August 12, 2019
+
+- **[Synopsis](#synopsis)**
+- **[Bugs resolved](#bugs)**
+- **[Credits](#credits)**
+- **[Feedback](#feedback)**
+
+## <a name="synopsis"></a>Synopsis
+
+| *Does this version...?*                                         |         |
+|:--------------------------------------------------------------- |:-------:|
+| Fix security vulnerabilities?                                   |   no    |
+| Change the database schema?                                     |   no    |
+| Alter the API?                                                  |   no    |
+| Require attention to configuration options?                     |   no    |
+| Fix problems installing or upgrading to a previous version?     |   no    |
+| Introduce features?                                             |   no    |
+| **Fix bugs?**                                                   | **yes** |
+
+## <a name="bugs"></a>Bugs resolved
+
+- **_Custom Search_: Resolve PHP fatal error ([dev/core#1175](https://lab.civicrm.org/dev/core/issues/1175): [#15009](https://github.com/civicrm/civicrm-core/pull/15009))**
+- **_Menus_: Menubar may not show up after clearing cache ([dev/core#1182](https://lab.civicrm.org/dev/core/issues/1182): [#15022](https://github.com/civicrm/civicrm-core/pull/15022))**
```

Comment:
Side note: I changed the categories (e.g. from "Regression" to "Custom Search" and "Menus").

The general purpose of a category is to let the reader/skimmer quickly determine if the patch involves part of the app that is relevant to them. (*Ex: Some CiviContribute users won't care if CiviCase changes; and vice-versa.*) In the context of patch-releases (*where selection-criteria are heavily tilted towards regressions*), it doesn't convey much to say "Regression" on every item.

Naming a specific subsystem ("CiviContribute" or "CiviMail") or artifact ("New Mailing"; " Contribution Detail Report") is more meaningful.